src/javascript/modules/tasks.js

```diff
@@ -0,0 +1,29 @@
+export default class Task {
+  constructor(name, dueDate = 'No date') {
+    this.name = name;
+    this.dueDate = dueDate;
+  }
```

Comment:
A todo item should contain the following fields
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/todo_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/todo-list)):
* A title
* A description
* A due date
* A priority

I was not prompted to enter most of this field in an attempt to create a new task.